certbot-apache/certbot_apache/parser.py

```diff
@@ -60,9 +60,12 @@ def __init__(self, aug, root, vhostroot, version=(2, 4)):
         # Set up rest of locations
         self.loc.update(self._set_locations())
 
-        # Must also attempt to parse virtual host root
-        self._parse_file(self.vhostroot + "/" +
-                         constants.os_constant("vhost_files"))
+        # Must also attempt to parse virtual host root, however
+        # omit sites-available for debian / ubuntu
+        if (not self.vhostroot.endswith("sites-available") and
+                not constants.os_constant("handle_sites")):
```

Comment:
Should we be calling `parse_file()` here at all anymore? If I understand correctly, Augeas already automatically parses any files included in an Apache configuration. With this in mind, why do we need to parse anything here manually?

If we need this call for some reason, is the logic here correct? Assuming `constants.py` doesn't change, we now don't parse the vhost directory if it ends in `*sites-available` or we're on Debian. What if a user has modified their main Apache config file to include `sites-available/*`? Should we be parsing the file then?